src/Exception/Cms/Event/EventNotFoundException.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types=1);
+
+namespace BalticRobo\Api\Exception\Cms\Event;
```

Comment:
Exceptions aren't assigned to CMS or something, just to domain.

Your domain is `Event`.